dev_requirements/requirements-tests.in

```diff
@@ -1,2 +1,3 @@
 pytest
 pytest-asyncio
+bo4e==0.6.1
```

Comment:
```suggestion
bo4e==0.6.1
# we're using this pinned version of bo4e because there's no version which contains this fix:
# https://github.com/bo4e/BO4E-python/commit/3fd45539edd7d5bfdca990775244bf31007a5ca7
```